examples/network-hhmodel/loader.rs

```diff
@@ -0,0 +1,85 @@
+use std::path::Path;
+
+use csv::Reader;
+use ixa::context::Context;
+use ixa::define_person_property;
+use ixa::{ContextPeopleExt, PersonId};
+use serde::Deserialize;
+use std::fs::File;
+
+#[derive(Deserialize, Copy, Clone, PartialEq, Eq, Debug, Hash)]
+pub enum AgeGroupValue {
+    U5,
+    U18,
+    Adult,
+    Old,
+}
+
+#[derive(Deserialize, Copy, Clone, PartialEq, Eq, Debug, Hash)]
+pub enum SexValue {
+    Female,
+    Male,
+}
+
+#[derive(Deserialize, Debug)]
+struct PeopleRecord {
+    id: u16,
+    age_group: AgeGroupValue,
+    sex: SexValue,
+    household_id: u16,
+}
+
+define_person_property!(Id, u16);
+define_person_property!(AgeGroup, AgeGroupValue);
+define_person_property!(Sex, SexValue);
+define_person_property!(HouseholdId, u16);
+
+fn create_person_from_record(context: &mut Context, record: &PeopleRecord) -> PersonId {
+    context
+        .add_person((
+            (Id, record.id),
+            (AgeGroup, record.age_group),
+            (Sex, record.sex),
+            (HouseholdId, record.household_id),
+        ))
+        .unwrap()
+}
+
+pub fn open_csv(file_name: &str) -> Reader<File> {
+    let current_dir = Path::new(file!()).parent().unwrap();
+    let file_path = current_dir.join(file_name);
+    csv::Reader::from_path(file_path).unwrap()
+}
+
+pub fn init(context: &mut Context) -> Vec<PersonId> {
+    // Load csv and deserialize records
+    let mut reader = open_csv("synthetic_households_us.csv");
```

Comment:
As discussed can we reduce the size of the example populations for the purposes of landing this? Alternatively, we could put this all into a separate repo, but unless we're planning on actively using/maintaining it i think it's probably better here.